src/middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export const config = {
+  matcher: '/((?!maintenance|_next).*)',
```

Comment:
```suggestion
  // NOTE: _nextも除外しているのは、内部処理が動かなくなってしまうため
  matcher: '/((?!maintenance|_next).*)',
```

のちのち良かれと思ってリファクタリングしちゃう可能性がちょっとあるかもだから↑入れとく？
`commit suggestion` ボタンで取り込めるはず